util/options_helper.cc

```diff
@@ -200,6 +201,31 @@ bool ParseBlockBasedTableIndexType(const std::string& type,
   return true;
 }
 
+bool SerializeEncodingType(
+  const EncodingType& type, std::string* value) {
+  switch (type) {
+    case EncodingType::kPlain:
+      *value = "kPlain";
```

Comment:
Maybe call it "prefix" and "plain". The common string is used in line 207 and 220. Would prefer defining a const string and use it in both places to avoid mistakes in the future.